src/components.jsx/ArticlesList.jsx

```diff
@@ -0,0 +1,32 @@
+import { SimpleGrid } from "@chakra-ui/react";
+import ArticleCard from "./ArticleCard";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+
+const ArticlesList = () => {
+  const topic = useParams();
+  const [currentArticles, setCurrentArticles] = useState([]);
+  useEffect(() => {
+    fetch(`https://project-nc-news-xdpp.onrender.com/api/articles`)

```

Comment:
handle requests in a separate utils file and use axios to remove the need to turn the incoming data to json before handling